@navikt/core/tokens/darkside/util.ts

```diff
@@ -34,17 +34,42 @@ export const globalColorScales = [
 
 export type GlobalColorScale = (typeof globalColorScales)[number];
 
-type TokenTypes = "color" | "global-color" | "global-radius" | "global-spacing";
+export const tokenTypes = {
+  color: "color",
+  "global-color": "global-color",
+  "global-radius": "global-radius",
+  "global-spacing": "global-spacing",
+} as const;
+
+export type TokenTypes = (typeof tokenTypes)[keyof typeof tokenTypes];
+
+export const tokenGroupLookup = {
+  background: "background",
+  border: "border",
+  text: "text",
+  contrast: "contrast",
+} as const;
+
+export type TokenGroup =
+  | GlobalColorRoles
+  | (typeof tokenGroupLookup)["background"]
+  | `${(typeof tokenGroupLookup)["background"]}.${GlobalColorRoles}`
+  | (typeof tokenGroupLookup)["text"]
+  | `${(typeof tokenGroupLookup)["text"]}.${GlobalColorRoles}`
+  | (typeof tokenGroupLookup)["border"]
+  | `${(typeof tokenGroupLookup)["border"]}.${GlobalColorRoles}`
+  | (typeof tokenGroupLookup)["border"]
+  | (typeof tokenGroupLookup)["contrast"];
```

Comment:
What's the point with that object? Suggestion:
```suggestion
export type TokenGroupLookup = "background" | "border" | "text" | "contrast"; // TODO: Find better name

export type TokenGroup =
  | GlobalColorRoles
  | TokenGroupLookup
  | `background.${GlobalColorRoles}`
  | `text.${GlobalColorRoles}`
  | `border.${GlobalColorRoles}`;
```
Or to reduce duplication:
```suggestion
export type TokenGroupLookup = "background" | "border" | "text" | "contrast"; // TODO: Find better name
type GroupWithRoles = Exclude<TokenGroupLookup, "contrast">; // TODO: Find better name

export type TokenGroup =
  | GlobalColorRoles
  | TokenGroupLookup
  | `${GroupWithRoles}.${GlobalColorRoles}`;
```
Maybe something similar can be done with `tokenTypes` as well.

Reply:
Fixed 🚀